client/src/components/RatingsAndReviews/Review.jsx

```diff
@@ -1,22 +1,50 @@
-import React from 'react';
+import React, {useState} from 'react';
 
 const Review = ({review}) => {
+
+  const [showFullBody, setShowFullBody] = useState(false);
+  const [bigPhotoSrc, setBigPhotoSrc] = useState("");
+  const [showModal, setShowModal] = useState(false);
+
+  // const openPhoto = (clickedPhoto) => {
+  //   const photo = document.getElementById('rr-photoModal');
+  //   const bigPhoto = document.getElementById('fullResolution');
+  //   bigPhoto.src = clickedPhoto;
+  //   photo.classList.add('rr-modal-visible');
+  // }
+
+  // const closePhoto = () => {
+  //   const photo = document.getElementById('rr-photoModal');
+  //   photo.classList.remove('rr-modal-visible');
+  // }
+
+  const toggleShowFullBody = () => {
+    setShowFullBody(!showFullBody);
```

Comment:
When updating state based on previous state I would recommend using an updater function. 
See https://react.dev/reference/react/useState#updating-state-based-on-the-previous-state